.github/workflows/ubuntu-20.04.yml

```
@@ -1,7 +1,11 @@
 ---
 name: build-ubuntu-20.04
 
-on: [push]
+on:
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)